content/projects/gsoc/2022/projects/plugin-health-scoring-system.adoc

```diff
@@ -25,20 +25,31 @@ links:
 
 
 === Abstract
-
-(to be described)
+This project aims to introduce a metric system to calculate the health score of each plugin within the Jenkins ecosystem and reflect the final scores on the Plugin Site for the plugin maintainers and users.
+The health score of each plugin would be shown on the UI with a detailed report of strengths and areas of improvement, making it easier for maintainers to maintain their plugins, and for users to decide if they should install/use a plugin.
 
 === Rationale
-
-(to be described)
+Plugin maintenance is an extremely important phase of a plugin’s lifecycle.
+Keeping a plugin current with the ongoing development of Jenkins is key for a smooth operation.
+Keeping it current can require user interface improvements, dependency updates, and more.
+Measurements of the plugin’s "health" will help maintainers as they decide how to invest their development time.
+Measurements of the plugin’s "health" will help users to decide if they should install and use a plugin.
+Plugins with a lower health score may be less likely to meet the user’s reliability needs.
+The plugin health score attempts to provide an accurate picture of how much care and help a plugin needs and makes it easier for every interested potential contributor to have an in-depth look at how they can fulfill those needs based on their expertise.
+This score also allows users to make a conscious decision before installing and/or using a plugin.
```

Comment:
This seems redundant given one of the previous points
> Measurements of the plugin’s "health" will help users to decide if they should install and use a plugin.

Reply:
Yes, I've merged these 2 sentences into one.